llvmlite/binding/newpassmangers.py

```diff
@@ -0,0 +1,283 @@
+from ctypes import c_bool, c_int
+from llvmlite.binding import ffi
+
+
+def create_new_module_pass_manager():
+    return NewModulePassManager()
+
+
+def create_new_function_pass_manager():
+    return NewFunctionPassManger()
+
+
+def create_pass_builder(tm, pto):
+    return PassBuilder(tm, pto)
+
+
+def create_pipeline_tuning_options(opt_level=2, size_level=0):
+    return PipelineTuningOptions(opt_level=opt_level, size_level=size_level)
+
+
+class NewModulePassManager(ffi.ObjectRef):
+
+    def __init__(self, ptr=None):
+        if ptr is None:
+            ptr = ffi.lib.LLVMPY_CreateNewModulePassManager()
+        super().__init__(ptr)
+
+    def run(self, module, pb):
+        ffi.lib.LLVMPY_RunNewModulePassManager(self, pb, module)
+
+    def addVerifier(self):
+        ffi.lib.LLVMPY_AddVerifierPass(self)
+
+    def add_aa_eval_pass(self):
+        ffi.lib.LLVMPY_AddAAEvalPass_module(self)
+
+    def add_simplify_cfg_pass(self):
+        ffi.lib.LLVMPY_AddSimplifyCFGPass_module(self)
+
+    def add_loop_unroll_pass(self):
+        ffi.lib.LLVMPY_AddLoopUnrollPass_module(self)
+
+    def add_loop_rotate_pass(self):
+        ffi.lib.LLVMPY_AddLoopRotatePass_module(self)
+
+    def add_instruction_combine_pass(self):
+        ffi.lib.LLVMPY_AddInstructionCombinePass_module(self)
+
+    def add_jump_threading_pass(self, threshold=-1):
+        ffi.lib.LLVMPY_AddJumpThreadingPass_module(self, threshold)
+
+    def _dispose(self):
+        ffi.lib.LLVMPY_DisposeNewModulePassManger(self)
+
+
+class NewFunctionPassManger(ffi.ObjectRef):
+
+    def __init__(self, ptr=None):
+        if ptr is None:
+            ptr = ffi.lib.LLVMPY_CreateNewFunctionPassManager()
+        super().__init__(ptr)
```

Comment:
Always creating a new underlying object, as above:

```suggestion
    def __init__(self):
        super().__init__(ffi.lib.LLVMPY_CreateNewFunctionPassManager())
```